modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/insertbutton/plugin.js

```diff
@@ -152,13 +164,16 @@
 				})
 			);
 
-			instance._toolbar.addItem(
+			this._toolbar.addItem(
 				'table',
 				new CKEDITOR.ui.balloonToolbarButton({
-					click() {
-						instance._toolbar.hide();
 
-						const tableToolbar = instance._createTableToolbar();
+					// Thanks specs gods of ES6 for implement such useful functionallity
+
+					click: () => {
```

Comment:
I needed to use an arrow function here to provide a `this` to be used internally.